Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml

```diff
@@ -52,6 +52,14 @@ configuration:
   defaultvalue: "25"
   type: 0
   required: false
+- display: Fetch Incident Cases
+  name: fetch_incident_cases
+  defaultvalue: Case Per Anomaly
```

Comment:
this changes the current behavior, which was one incident for all anomalies - are we sure we want that to be the default value? if so, lets document this in the release notes as breaking change

Reply:
Yes,Default value as "Case Per Anomaly"

Reply:
ok so please mention this in the release notes as this is a breaking change